### PR TITLE
Translation of javascript chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of [#586](https://github.com/HTTPArchive/almanac.httparchive.org/issues/586)

I coluld not find `../2021/javascript/#async-and-defer` on line 210

I've also updated src/content/en/2022/javascript.md
Please confirm it too.

### CODE REVIEW

**Comment by tunetheweb** [comment]
> I coluld not find `../2021/javascript/#async-and-defer` on line 210

This should be `../2021/javascript#async-and-defer` (without the final slash) for English and `./2021/javascript#asyncとdefer` for Japanese.

File: src/content/ja/2022/javascript.md [hunk 1]
```diff
@@ -0,0 +1,1074 @@
...
+function n(n,r){return n+r}
+```
+
+バンドルは、ソースコードを最適化し、実稼働環境でのパフォーマンスを向上させるための重要な役割を担っています。
+
+JavaScriptのバンドルに関しては豊富な選択肢がありますが、よく頭に浮かぶのは<a hreflang="en" href="https://webpack.js.org/">webpack</a>でしょう。幸いなことに、webpackの生成するJavaScriptにはいくつかのシグネチャ（たとえば `webpackJsonp`）が含まれており、ウェブサイトのプロダクションJavaScriptがwebpackを使ってバンドルされているかどうかを検出することが可能です。
+
+{{ figure_markup(
+    image="webpack-rank.png",
+    caption="webpack-bundled JavaScriptを使用しているページの順位。",
```
Comment (tunetheweb) [inline src/content/ja/2022/javascript.md:108]: Should the "-bundled" bit be translated? This means "JavaScript bundled through webpack"

File: src/content/ja/2022/javascript.md [hunk 2]
```diff
@@ -0,0 +1,1074 @@
...
+    sheets_gid="1329160999",
+    sql_file="usage_of_webpack_by_rank.sql"
+  )
+}}
+
+もっとも普及している1,000のWebサイトのうち、17%がバンドルとしてwebpackを使用しています。HTTP Archiveがクロールするもっとも多いページの多くは、ソースコードのバンドルと最適化にwebpackを使用している知名度の高いeコマースサイトである可能性が高いため、これは理にかなった結果です。それでも、HTTP Archiveのデータセットに含まれる全ページの5%がwebpackを使用しているという事実は、重要な統計データです。しかし、使用されているバンドルはwebpackだけではありません。
+
+{{ figure_markup(
+    image="parcel-rank.png",
+    caption="Parcel-bundled JavaScriptを使用しているページの順位。",
```
Comment (tunetheweb) [inline src/content/ja/2022/javascript.md:120]: Same question here

**Review by tunetheweb — COMMENTED** [review]
LGTM!

Did some minor tweaks (FYI for the Mozilla links we are no longer including the language as Mozilla redirects as needs be so saves us some effort).

Two small questions and then can merge and release!

File: src/content/ja/2022/javascript.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2022/javascript.md]: Thanks I translate.

File: src/content/ja/2022/javascript.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2022/javascript.md]: Thanks I translate.

**Comment by ksakae1216** [comment]
@tunetheweb I fixed. Please review.

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**